addons/purchase_stock/tests/test_purchase_lead_time.py

```diff
@@ -239,3 +239,72 @@ def test_merge_po_line_2(self):
         self.assertEqual(len(po_line), 2, 'the purchase order lines are not merged')
         self.assertEqual(po_line[0].product_qty, 10, 'the purchase order line has a wrong quantity')
         self.assertEqual(po_line[1].product_qty, 5, 'the purchase order line has a wrong quantity')
+
+    def test_merge_po_line_3(self):
+        """Chage that merging po line for same procurement is done depending on
+        propagate_date and description"""
+
+        # create a product with manufacture route
```

Comment:
what are you testing? you don't even use the description field you added on the move?

Reply:
done